packages/docs/getting-started/running-a-validator.md

```diff
@@ -50,6 +50,15 @@ $ export CELO_VALIDATOR_GROUP_ADDRESS=<YOUR-VALIDATOR-GROUP-ADDRESS>
 $ export CELO_VALIDATOR_ADDRESS=<YOUR-VALIDATOR-ADDRESS>
 ```
 
+In order to register the validator later on, generate a "proof of possession" - a signature proving you know your validator's BLS private key. Run this command:
```

Comment:
Should this be removed for now?

Reply:
Hm, good question. It will still perform correctly, and that's the way I expect it to be moving forward anyway. We need a string of appropriate length. Maybe worth leaving it?

Reply:
@asaj @kevjue what do you think?

Reply:
My suggestion is to remove, and add later.